src/schemathesis/schemas.py

```diff
@@ -232,8 +244,14 @@ def get_full_path(self, path: str) -> str:
 
     def get_all_endpoints(self) -> Generator[Endpoint, None, None]:
         try:
-            paths = self.raw_schema["paths"]  # pylint: disable=unsubscriptable-object
+            if self.operationId:
+                # use the operationId to uniquely identify an operation
+                paths = self.get_endpoint_by_operationId()
```

Comment:
The name seems misleading to me - I would expect that there will be a single endpoint returned instead of paths that I assume (from the variable name) may contain multiple values.

Reply:
Deleted it,filtering will be run later in the code